crm_multicompany_reporting_currency/models/crm_lead.py

```diff
@@ -79,5 +79,6 @@ def _compute_amount_multicompany_reporting_currency(self):
             if record.company_currency == record.multicompany_reporting_currency_id:
                 to_amount = record.expected_revenue
             else:
-                to_amount = record.expected_revenue / record.currency_rate
+                inverse_rate = 1.0 / record.currency_rate
+                to_amount = record.expected_revenue / inverse_rate
```

Comment:
Mathematically, to divide by the inverse rate is equivalent to multiplying by the original rate, so this snippet of code is equivalent to:
```suggestion
                to_amount = record.expected_revenue * record.currency_rate
```
which (I believe) is easier to read

Reply:
I'll check @SilvioC2C, thanks